functions.yml

```diff
@@ -153,17 +162,21 @@ etl:
           [Row(calls=[-1, -1])]
       args:
         - name: probabilities
-          doc: Probabilities
+          doc: The array of probabilities to convert
         - name: numAlts
-          doc: The number of alts
+          doc: The number of alternate alleles
         - name: phased
-          doc: Whether the probabilities are phased or not
+          doc: Whether the probabilities are phased. If phased, we expect one ``2 * numAlts`` values
+            in the probabilities array. If unphased, we expect one probability per possible genotype.
         - name: threshold
-          doc: The minimum probability to include
+          doc: The minimum probability to make a call. If no probability falls into the range 
+            of ``[0, 1 - threshold]`` or ``[threshold, 1]``, a no-call (represented by ``-1``s) will be emitted.
```

Comment:
The `-1` didn't render properly because it's immediately followed by an `s`

Reply:
Fixed